cloudfour/cloudfour.com-patterns
# Pattern: message

## Overview

Introduces a (fullwidth) message pattern ideal for offline network status notifications.

This PR is open to both design and development feedback. 

## Screenshots

![Screen Shot 2020-04-30 at 11 38 10 AM](https://user-images.githubusercontent.com/1427548/80746671-21545c80-8ad7-11ea-93bd-f93601315c43.png)

Close button `:hover` style

![message-with-close-hover](https://user-images.githubusercontent.com/1427548/81018430-5afbcf00-8e19-11ea-8695-ade00e30ff3c.gif)


Close button with active `:focus`

![Screen Shot 2020-05-04 at 3 07 59 PM](https://user-images.githubusercontent.com/1427548/81018295-1839f700-8e19-11ea-8237-811af5dcee91.png)

## Testing

1. View Netlify deploy preview. 
https://deploy-preview-604--cloudfour-patterns.netlify.com/?path=/docs/components-message--message

2. Confirm all Storybook accessibility checks pass.

---

Issue #510
